packetbeat/protos/cassandra/_meta/fields.yml

```diff
@@ -2,10 +2,20 @@
   title: "Cassandra"
   description: Cassandra v4/3 specific event fields.
   fields:
+    - name: no_request
+      type: alias
+      path: cassandra.no_request
+      migration: true
+
```

Comment:
Nit: I personally like to move the aliases at the end of the file, so that the generated doc starts with the good stuff, and ends with the aliases.

Reply:
Also, I agree with moving this top level field to `cassandra.no_request` 👍